modules/dynamic-plug-in-api.adoc

```diff
@@ -988,7 +991,7 @@ Provides apiVersion for a k8s model.
 [discrete]
 == `getGroupVersionKindForResource`
 
-Provides a group, version, and kind for a resource. It returns the group, version, kind for the provided resource. If the resource does not have an API group, group "core" will be returned.If the resource has an invalid apiVersion then it'll throw Error.
+Provides a group, version, and kind for a resource. It returns the group, version, kind for the provided resource. If the resource does not have an API group, group "core" will be returned. If the resource has an invalid apiVersion, then it will throw an Error.
```

Comment:
Are we allowed to change the wording like this for API docs? I know we are unable to change the wording on the rest_API docs. I am cool with this if we are good to change it.

Reply:
Per our Slack chat and just to duplicate here, I believe REST API and CLI docs are special because of the generation we do from the upstream source, and should therefore be making any tweaks to the content upstream (to be re-generated for downstream docs).

I definitely don't want to adjust the meaning of other API or CLI reference material, but I think small tweaks for typos / grammar / formatting-requirements are fair game.